### ISSUE TITLE
[FEATURE] Allow training of multiple datasets at once

### ISSUE THREAD

**Issue by AlmightyYakob** [VIAME/VIAME-Web#391]
I'm not too sure of the use case, @mattdawkins please comment the specifics.

**Comment by mattdawkins** [comment]
This is very high priority, at least the basic form of it (training on multiple sequences, where each sequence is an annotated video or image sequence in the same folder).

The next level up would be training on multiple sequences in different folders which is also good to have though slightly lower priority.

Training on one sequence isn't that useful, only for some small models and for annotation assist models.

**Comment by AlmightyYakob** [comment]
> This is very high priority, at least the basic form of it (training on multiple sequences, where each sequence is an annotated video or image sequence in the same folder).

The way we currently have things implemented, this would probably amount to us adding the functionality to run training on a folder, where all immediate subdirectories are image sequences / videos. Does that accomplish what you're talking about? Or are you talking about something else?

> The next level up would be training on multiple sequences in different folders which is also good to have though slightly lower priority.

Would this be covered by the case I mentioned above? Or would this be more like jumping around the browser and selecting datasets in various locations to add to a training "batch"?

**Comment by BryonLewis** [comment]
I have a first pass working for this right now in a Draft PR.  After some discussion with Brandon, I'll be changing this to use the new manifest structure for training as soon as it available.

This is at least a working version of training that I'll be doing some testing on to make sure it functions properly and that I didn't mess up too much stuff.

**Comment by BryonLewis** [comment]
The manifest file is in progress and will be integrated into the draft PR once it is ready for testing.
